client/assets/stylesheets/_wp-components-overrides.scss

```diff
@@ -35,8 +39,9 @@
 	&:disabled,
 	&.disabled {
 		color: var(--color-neutral-20);
-		background-color: var(--color-surface);
+		background-color: var(--gray-gray-5, #dcdcde);
 		border-color: var(--color-neutral-5);
+		opacity: 1;
```

Comment:
```suggestion
```
Check my work but I think this gets applied already via wordpress/components (on disabled + primary button classes.)